.cminx-config.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 CMakePP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
There's already a CMinx config in `docs/source/config.yml` with options we've been using. One of the ones missing from this config is the module path separator, CMinx defaults to `.`, but we've found `/` makes more sense given the syntax for importing modules.

If this config will only be used in the action, I also suggest copying the `logging` config from the default config and changing the level to DEBUG so if the action fails we can see more clearly what's going on:

https://github.com/CMakePP/CMinx/blob/6ec67b6278502271a4fe7f78ada12d739ae05539/src/cminx/config_default.yaml#L133

Reply:
We will use the existing config in `docs/source/config.yml`.